kubernetes/config-maps/nginx/ssl.conf

```diff
@@ -0,0 +1,26 @@
+# Disable SSLv3
+ssl_protocols              TLSv1 TLSv1.1 TLSv1.2;
+
+# Enable server-side protection
+ssl_prefer_server_ciphers  on;
+ssl_ciphers                "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
+
+# Specify curve P-384
+ssl_ecdh_curve             secp384r1;
+
+# Improve ssl performence
+ssl_session_cache          shared:SSL:10m;
+ssl_session_timeout        5m;
```

Comment:
Add a link to this site in a comment: https://vincent.bernat.im/en/blog/2011-ssl-session-reuse-rfc5077